packages/analyzer/src/utils/jsdoc.js

```diff
@@ -7,8 +7,8 @@ export function handleJsDocType(type) {
 }
 
 export function normalizeDescription(desc) {
-  if (desc.startsWith?.('- ')) {
+  if (desc.startsWith('- ')) {
```

Comment:
Why did we remove the conditional here?

Reply:
AFAIK the lack of the conditional is what's causing the crash in the [playground](https://custom-elements-manifest.netlify.app/?source=CmNsYXNzIE15RWxlbWVudCBleHRlbmRzIEhUTUxFbGVtZW50IHsKICAgIHN0YXRpYyBnZXQgb2JzZXJ2ZWRBdHRyaWJ1dGVzKCkgewogICAgICAgIHJldHVybiBbJ2Rpc2FibGVkJ107CiAgICB9CgogICAgc2V0IGRpc2FibGVkKHZhbCkgewogICAgICAgIHRoaXMuX19kaXNhYmxlZCA9IHZhbDsKICAgIH0KICAgIGdldCBkaXNhYmxlZCgpIHsKICAgICAgICByZXR1cm4gdGhpcy5fX2Rpc2FibGVkOwogICAgfQoKICAgIC8qKgogICAgICogR2VuZXJhdGVzIGEgVVJMIGZvciBhbnkgYXNzZXQgaW4geW91ciBNZWRpYSBsaWJyYXJ5LgogICAgICogQGZ1bmN0aW9uIENsb3VkaW5hcnkjdXJsCiAgICAgKiBAcGFyYW0ge09iamVjdH0gW29wdGlvbnNdIC0gVGhlIHtAbGluayBUcmFuc2Zvcm1hdGlvbn0gcGFyYW1ldGVycyB0byBpbmNsdWRlIGluIHRoZSBVUkwuCiAgICAgKiBAcmV0dXJuIHtzdHJpbmd9IFRoZSBtZWRpYSBhc3NldCBVUkwuCiAgICAgKi8KICAgIHVybChwdWJsaWNJZCkgewogICAgICAgIHZhciBvcHRpb25zID0gYXJndW1lbnRzLmxlbmd0aCA%2BIDEgJiYgYXJndW1lbnRzWzFdICE9PSB1bmRlZmluZWQgPyBhcmd1bWVudHNbMV0gOiB7fTsKICAgICAgICByZXR1cm4gdXJsX3VybChwdWJsaWNJZCwgb3B0aW9ucywgdGhpcy5jb25maWcoKSk7CiAgICB9CgogICAgZmlyZSgpIHsKICAgICAgICB0aGlzLmRpc3BhdGNoRXZlbnQobmV3IEV2ZW50KCdkaXNhYmxlZC1jaGFuZ2VkJykpOwogICAgfQp9CgpjdXN0b21FbGVtZW50cy5kZWZpbmUoJ215LWVsZW1lbnQnLCBNeUVsZW1lbnQpOwo%3D&library=null)

Reply:
What happens is that `desc` in this case turns out to be an array of comments, which doesnt have a `startsWith` method on it. But now (in the change above), we flatten the array and turn it into a string.

Reply:
Ahh gotcha. Missed that piece of code. All good 🚀